Makefile

```diff
@@ -17,5 +17,5 @@ clean:
 
 .PHONY: release
 release:
-	go get github.com/mitchellh/gox
-	gox -ldflags="$(FLAGS)" -output="build/{{.Dir}}-{{.OS}}-{{.Arch}}" -osarch="linux/amd64 windows/amd64" .
+	go install github.com/mitchellh/gox
+	gox -ldflags="$(FLAGS)" -output="build/{{.Dir}}-{{.OS}}-{{.Arch}}" -osarch="linux/amd64 windows/amd64 linux/arm64 darwin/arm64" .
```

Comment:
Add `darwin/amd64` for completeness?

`gox -os="darwin linux windows" -arch="amd64 arm64"` is a slightly more succinct way of getting the same result.

Reply:
Updated in https://github.com/buildkite/ecs-run-task/pull/68/commits/2f30dc8828557647764c44d231313145373b01cf